### PR TITLE
LPS-129996 Create flag to display a new UI of Workflow

### PR DESCRIPTION
**Pull request by MarinhoFeliphe**
Story: [LPS-129996](https://issues.liferay.com/browse/LPS-129996)

### CODE REVIEW

File: modules/dxp/apps/portal-workflow/portal-workflow-kaleo-designer-web/src/main/resources/META-INF/resources/designer/edit_workflow_definition.jsp [hunk 1]
```diff
@@ -0,0 +1,23 @@
...
+ *
+ */
+--%>
+
+<%@ include file="/designer/init.jsp" %>
+
+<div>
+	<react:component
+		module="js/index.es"
+	/>
```
Comment (rafaprax) [inline modules/dxp/apps/portal-workflow/portal-workflow-kaleo-designer-web/src/main/resources/META-INF/resources/designer/edit_workflow_definition.jsp:22]: Hey @MarinhoFeliphe, is there a `js/index.es` file?

File: modules/dxp/apps/portal-workflow/portal-workflow-kaleo-designer-web/src/main/java/com/liferay/portal/workflow/kaleo/designer/web/internal/portlet/display/context/KaleoDesignerDisplayContext.java [hunk 2]
```diff
@@ -771,6 +774,14 @@ private String _buildErrorLink(String messageKey, PortletURL portletURL) {
 			});
 	}
 
+	private String _getJPSPath() {
+		if (_ffKaleoDesignerConfigurationActivator.newKaleoDesignerEnabled()) {
```
Comment (rafaprax) [inline modules/dxp/apps/portal-workflow/portal-workflow-kaleo-designer-web/src/main/java/com/liferay/portal/workflow/kaleo/designer/web/internal/portlet/display/context/KaleoDesignerDisplayContext.java:778]: Hey @MarinhoFeliphe , rename to `_getJspPath`.

File: modules/dxp/apps/portal-workflow/portal-workflow-kaleo-designer-web/src/main/java/com/liferay/portal/workflow/kaleo/designer/web/internal/portlet/action/BaseKaleoDesignerMVCActionCommand.java [hunk 3]
```diff
@@ -220,6 +224,14 @@ protected void setRedirectAttribute(
 	@Reference
 	protected WorkflowDefinitionManager workflowDefinitionManager;
 
+	private String _getJPSPath() {
+		if (ffKaleoDesignerConfigurationActivator.newKaleoDesignerEnabled()) {
```
Comment (rafaprax) [inline modules/dxp/apps/portal-workflow/portal-workflow-kaleo-designer-web/src/main/java/com/liferay/portal/workflow/kaleo/designer/web/internal/portlet/action/BaseKaleoDesignerMVCActionCommand.java:228]: Hey @MarinhoFeliphe , rename to _getJspPath.

**Review by rafaprax — CHANGES_REQUESTED** [review]

**Comment by MarinhoFeliphe** [comment]
ci:test:sf

**Comment by MarinhoFeliphe** [comment]
ci:test:relevant

**Comment by MarinhoFeliphe** [comment]
ci:stop

**Comment by MarinhoFeliphe** [comment]
ci:test:relevant

**Comment by rafaprax** [comment]
Just started reviewing :)

**Comment by rafaprax** [comment]
resent here https://github.com/liferay-workflow/liferay-portal/pull/535

**Pull request closed without merging**